packages/vuepress-theme-odyssey/templates/DocsTemplateIndex.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <article class="docs-index">
+    <DocsPageHeader
+      :title="$page.frontmatter.title"
+      :lead="$page.frontmatter.lead"
+      :variant="$page.frontmatter.id"
+    />
+    <DocsCardGroup
+      tag="ul"
+      :variant="$page.frontmatter.id === 'base' && '2col'"
+    >
+      <DocsCard
+        v-for="(page, index) in content"
+        :key="index"
+        tag="li"
+        variant="shadow"
+        class="docs-index--card"
+      >
+        <template slot="header">
+          <!-- eslint-disable -->
+        <div
+          aria-hidden
+          class="docs-card--header-image"
+          v-html="
+            require(`!html-loader!../../docs/.vuepress/public/images/coin-${page.id || 'fpo'}.svg`)
+          "
+        />
+          <!-- eslint-enable -->
+          <h2>{{ page.title }}</h2>
+        </template>
+        <div class="docs-index--card-description">
+          <p>
+            {{ page.description }}
+          </p>
+        </div>
+        <template slot="footer">
+          <DocsLink :href="page.link"
+            >Learn more
+            <span class="u-visually-hidden"
+              >about {{ page.title }}</span
+            ></DocsLink
+          >
```

Comment:
Do we need to break these components apart in this way? It is super hard to read.

```suggestion
          <DocsLink :href="page.link">
            Learn more <span class="u-visually-hidden">about {{ page.title }}</span>
          </DocsLink>
```

Reply:
This is a good catch, I think my code editor did some weird format on save stuff (which I have since turned off)